backend/src/controllers/billingController.js

```diff
@@ -13,4 +13,30 @@ export default class BillingController {
       });
     });
   }
+
+  deleteBillable(billableId) {
+    return new Promise((resolve, reject) => {
+      const BillableModel = new Billable();
+
+      BillableModel.deleteBillable(billableId, (err, result) => {
+        if (err) {
+          reject({ error: err });
+        }
+        resolve(result);
+      });
+    });
+  }
+
+  saveBillable(billable) {
+    return new Promise((resolve, reject) => {
+      const BillableModel = new Billable();
+
```

Comment:
You need to construct the billable object like this: https://github.com/ubclaunchpad/labby/blob/main/backend/src/controllers/userController.js#L48